nav2_controller/README.md

```diff
@@ -0,0 +1,11 @@
+# Nav2 Controller
+
+The Nav2 Controller is an [Execution Module](../doc/requirements/requirements.md) that implements the `nav2_behavior_tree::FollowPathAction` interface.
+
+An execution module implementing the `nav2_behavior_tree::FollowPathAction` interface is responsible for generating command velocities for the robot, given the computed path from the planner module in `nav2_planner`. The nav2_controller package is designed to be loaded with plugins for path execution. The plugins need to implement functions in the virtual base class defined in the `local_planner` header file in `nav2_core` package.
+
+
+Currently available controller plugins are: DWB, and [TEB (dashing release)](https://github.com/rst-tu-dortmund/teb_local_planner/tree/dashing-devel).
+
+
+The nav2_controller node is a Lifecycle node.
```

Comment:
`nav2_controller` is the namespace while `ControllerServer` is the node.

Reply:
I am removing this line. Maybe somewhere we should state what all nodes are lifecycle enabled.